rpc/common/moc_common_computecommon.proto

```diff
@@ -237,6 +237,19 @@ enum SecurityType {
         CONFIDENTIALVM = 2;
 }
 
+enum SecurityEncryptionTypes {
+    SecurityEncryptionNone = 0;
+    NonPersistedTPM = 1;
```

Comment:
Note for other platforms. Stateless is NonManagementVTL. not NonPersistedTPM.
At the VM creation time you will need to determine which type to set based off of the platform.  Just makeing sure that this doesnt limit us.

Reply:
discussed offline, we will revisit renaming this in the future